src/Sulu/Bundle/MediaBundle/Media/ImageConverter/MediaImageExtractor.php

```diff
@@ -47,9 +47,18 @@ public function __construct(
         $this->videoThumbnail = $videoThumbnail;
     }
 
-    public function extract($resource)
+    public function extract($resource/*, string $resourceMimeType*/)
```

Comment:
do you think this is better than making the `$resourceMimeType` nullable? i think ide support and static code analysis would be better with a nullable parameter

Reply:
That would be a bc break as the class is not final and somebody could extend it.